starter/script.js

```diff
@@ -1 +1,97 @@
-// put code here!
+var alive = 1;
+while(alive===1){
+  var response = prompt('Welcome to Choose Your Own Adventure: Harry Potter\n\nA: You are Harry Potter, walking down a dark corridor at Hogwarts with Hermione and Ron.  You are searching for the final Horcrux.  There is a door on your left and you hear a noise behind it.  Do you open the door? (y/n)');
+  switch(true){
```

Comment:
there are some interesting unconventional uses of switch for sure, but I just want to mention that with switch it is more typical to pass some variable in as the argument to switch. One of the reasons for this is that switch is particularly well-suited to considering three distinct cases where only one will be true.

reference: http://stackoverflow.com/questions/2765981/javascript-switchtrue